### ISSUE TITLE
Many problems

### ISSUE THREAD

**Issue by bnff** [n1kobg/PERFORMANCE-BOOSTER#2]
Hello !
Thanks for this script ... 

I have tests, I have a lot of mistakes ... which can not write in the start menu ( run ctfmon.exe dont work)
I think with small modification and reorganization it can be very efficient !

**Comment by n1kobg** [comment]
Yes, thankx for the feedback. i already fixed the problems. Plsease read the main page:
https://n1kobg.blogspot.com/p/performanceboosterv3.html

This is where I make the changes first. Down bellow i try to describe the changes that worth mentioning.
I dont think you have a lot of "mistakes" :) If you ran it ion v1809. Unfortunately the restore making tool have to be enables manually 1st, then works as normal. I'll change it with a comand which will work 100% from the 1st time. I fixed the StartMenu. I use 3rd party StartMenu and dont care about Microsoft Universal apps. This is actually for me but if someone want to use it im happy to make the needed changes in order to work for others.
p.s,
And please dont  ask me why CHEF-KOCH put my guide in negatives. He thinks I have to leave Cortana enabled even if I dont use it. I leave my thought on every tweak, especially these considered as myth. The guy just reads, dont test. IT guys know the language better but most dont have a clue whats good for gaming when it comes to performance (I aim for maximum performance and dont care about visuals).
